pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.coverage.run]
+source_pkgs = ["imaplib2"]
+omit = ["*/tests/*"]
```

Comment:
Don't omit tests

Reply:
Done. Removed `omit` and added `source = ["imaplib2/tests"]`.